docs/guides/vpn/wireguard/client.md

```diff
@@ -54,8 +54,8 @@ For each new client, the following steps must be taken. For the sake of simplici
     sudo -i
     cd /etc/wireguard
 
-    bash "10.100.0." "fd08:4711::" "my_server_domain:47111" 2 "annas-android"
-    bash "10.100.0." "fd08:4711::" "my_server_domain:47111" 3 "peters-laptop"
+    bash /path/to/script "10.100.0." "fd08:4711::" "my_server_domain:47111" 2 "annas-android"
```

Comment:
```suggestion
    ./path/to/script.sh "10.100.0." "fd08:4711::" "my_server_domain:47111" 2 "annas-android"
```

Reply:
Yes, we need it. For direct invocation you need to run `chmod +x` first.

Reply:
I added the appropriate instructions.